### ISSUE TITLE
[Feature] A Hoi4-Inspired War System

### ISSUE THREAD

**Issue by gabbobler** [EarthMC/Issue-Tracker#379]
Hello, i'm gabbobler. I had an idea of a war system based on HOI4. So the idea is "**capitulation**" and having the warmonger (person declaring war) having to pay 50-100G to start a war, making it so wars cannot be declared by anyone. It would be cool to also have your citizens vote yes on the war.

**The Capitulation Mechanic**
So the idea is, your entire town's blocks and landmass will be calculated. Once you're at war, if your town is bombed to shreds and there isn't much left (let's say 1/4th of your entire nation is destroyed/griefed in a war), it will automatically come to a peace treaty. In this peace treaty, the warmonger will take anything he wants. (Note, peace will be available anytime except for the first 2 hours of the war). 

TL;DR : Make a system where the landmass of the nation is calculated and if your landmass is lower than a specific number you automatically lose and the winner gets to take anything.

**Comment by cat234** [comment]
Take all states
"France has capitulated"

**Comment by cat234** [comment]
No really, this would be cool.
But towny cannot do this, there isn't even a ting in towny to allow wars.

**Comment by BowShot118** [comment]
gl getting someone to program this tho

**Comment by gabbobler** [comment]
@BowShot118 
The difficult part would probably be the chunk calculation. The rest I assume is pretty straight forward.

**Comment by shirazmatas** [comment]
Seems hard to program, if you know anyone capable of it go ahead and recommend to develope a prototype. I think you'd have to expand your idea further though.

**Comment by Ultil0l** [comment]
That would be epic

**Comment by KarlOfDuty** [comment]
Put this on the towny issue tracker, not ours.
